ui/src/form.tsx

```diff
@@ -286,11 +282,12 @@ export const
   XInline = ({ model: m }: { model: Inline }) => (
     <XComponents
       items={m.items}
-      justify={m.justify || 'start'}
-      align={m.align || 'center'}
+      justify={m.justify}
+      align={m.align || m.direction === 'row' ? 'center' : undefined}
```

Comment:
```suggestion
      align={m.align || (m.direction === 'row' ? 'center' : undefined)}
```
Without the parentheses the condition results in wrong value a therefore alignment can result always being `center` in some cases - please, see the **align** demo in broken `inline.py` example.

Reply:
Good catch, thanks!